desktop_version/src/Render.cpp

```diff
@@ -1985,8 +1985,15 @@ void gamerender(void)
             game.activity_lastprompt.c_str()
         );
 
-        graphics.drawtextbox(16, 4, 36, 3, game.activity_r*act_alpha, game.activity_g*act_alpha, game.activity_b*act_alpha);
-        graphics.Print(5, 12, final_string, game.activity_r*act_alpha, game.activity_g*act_alpha, game.activity_b*act_alpha, true);
+        if (game.activity_r == 0 && game.activity_g == 0 && game.activity_b == 0)
+        {
+            graphics.bprint(game.activity_x - 11, game.activity_y + 8, final_string, 196*act_alpha, 196*act_alpha, (255 - help.glow)*act_alpha, true);
+        }
+        else
+        {
+            graphics.drawtextbox(game.activity_x, game.activity_y, 36, 3, game.activity_r*act_alpha, game.activity_g*act_alpha, game.activity_b*act_alpha);
+            graphics.Print(game.activity_x - 11, game.activity_y + 8, final_string, game.activity_r*act_alpha, game.activity_g*act_alpha, game.activity_b*act_alpha, true);
+        }
```

Comment:
It might be hard to spot (I've been strongly considering refactoring all the print callsites), but what's happening here is that the text in the prompt is always centered and thus the x-position doesn't matter (including transparent prompts), whereas the surrounding text box is not centered at all and thus the x-position affects it there.

I'm not sure what your intentions here are with regards to horizontal positioning, but this is at the very least an inconsistency.

Reply:
> I'm not sure it makes much difference. Either you have to rebase and fix conflicts on all of my work, or I have to rebase and fix conflicts on all of your work.

I meant that you could hold off on refactoring print callsites until after the localization branch is merged, so you can directly base the changes on that - as you already suggest is a possibility in the second paragraph of your comment.

> Besides, it's virtually guaranteed you'll run into conflicts when maintaining an unmerged fork for extremely long periods of time.

I know, I have already encountered and resolved a lot of conflicts, I'm just saying that the print lines would cause exceedingly high amounts of them so it'd be better to avoid those extra hours of concentrated staring at little details.